src/Microsoft.VisualStudio.SDK.Analyzers.Tests/DiagnosticVerifier.cs

```diff
@@ -30,7 +30,7 @@ public abstract class DiagnosticVerifier
 
         private static readonly ImmutableArray<string> VSSDKPackageReferences = ImmutableArray.Create(new string[]
         {
-            Path.Combine("Microsoft.VisualStudio.OLE.Interop", "7.10.6071", "lib", "Microsoft.VisualStudio.OLE.Interop.dll"),
+            Path.Combine("Microsoft.VisualStudio.OLE.Interop", "7.10.6071", "lib\\net11", "Microsoft.VisualStudio.OLE.Interop.dll"),
```

Comment:
This shouldn't be here. I struggled with this recently as well. I think one of our *internal* package feeds has a package with this version number and a net11 subfolder. But it's not in the package on nuget.org, so this would cause the tests to fail on appveyor.

Reply:
Ok I will revert this with the other change in this file